app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -107,6 +107,10 @@ def machine_id(feed)
       os_resource_for(feed).try(:properties).try { |prop| prop['Machine Id'] }
     end
 
+    def container_id(feed)
```

Comment:
I wonder if this class a proper place for such methods (although this addition repeats another method that has been already here)